spring-cloud-gcp-test/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>spring-cloud-gcp</artifactId>
+        <groupId>org.springframework.cloud</groupId>
+        <version>1.2.3.BUILD-SNAPSHOT</version>
+    </parent>
+
+
+    <artifactId>spring-cloud-gcp-test</artifactId>
+    <name>Spring Cloud GCP Test Support</name>
+    <description>Provides tools for testing Spring Cloud GCP projects</description>
+    <dependencies>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
```

Comment:
`test` scope?

Reply:
Not in this case -- the test project itself is meant to be imported as test scope, but the rule extends a JUnit class, so it needs to be compile, not test scope.

Reply:
I wonder if we should consider making these utilities independent of JUnit. People use different test frameworks.

Reply:
make it generic so people can reuse it sounds good. Can you take into account that may some people will be able to contribute creating rules for junit4 or extension for junit5 specifically? Those can be issues with `ideal for contribution` or `help wanted`. WDYT?

Reply:
That a good idea! We should do that.